src/gcp_scanner/scanner.py

```diff
@@ -299,104 +299,26 @@ def main():
-
-  parser.add_argument(
-      '-k',
-      '--sa-key-path',
-      default=None,
-      dest='key_path',
-      help='Path to directory with SA keys in json format')
-  parser.add_argument(
-      '-g',
-      '--gcloud-profile-path',
-      default=None,
-      dest='gcloud_profile_path',
-      help='Path to directory with gcloud profile. Specify -\
- to search for credentials in default gcloud config path'
-  )
-  parser.add_argument(
-      '-m',
-      '--use-metadata',
-      default=False,
-      dest='use_metadata',
-      action='store_true',
-      help='Extract credentials from GCE instance metadata')
-  parser.add_argument(
-      '-at',
-      '--access-token-files',
-      default=None,
-      dest='access_token_files',
-      help='A list of comma separated files with access token and OAuth scopes.\
-TTL limited. A token and scopes should be stored in JSON format.')
-  parser.add_argument(
-      '-rt',
-      '--refresh-token-files',
-      default=None,
-      dest='refresh_token_files',
-      help='A list of comma separated files with refresh_token, client_id,\
-token_uri and client_secret stored in JSON format.'
-  )
-  parser.add_argument(
-      '-s',
-      '--service-account',
-      default=None,
-      dest='key_name',
-      help='Name of individual SA to scan')
-  parser.add_argument(
-      '-p',
-      '--project',
-      default=None,
-      dest='target_project',
-      help='Name of individual project to scan')
-  parser.add_argument(
-      '-f',
-      '--force-projects',
-      default=None,
-      dest='force_projects',
-      help='Comma separated list of project names to include in the scan')
-  parser.add_argument(
-      '-c',
-      '--config',
-      default=None,
-      dest='config_path',
-      help='A path to config file with a set of specific resources to scan.')
-  parser.add_argument(
-      '-l',
-      '--logging',
-      default='WARNING',
-      dest='log_level',
-      choices=('INFO', 'WARNING', 'ERROR'),
-      help='Set logging level (INFO, WARNING, ERROR)')
-
-  args: argparse.Namespace = parser.parse_args()
-
-  if not args.key_path and not args.gcloud_profile_path \
-    and not args.use_metadata and not args.access_token_files\
-    and not args.refresh_token_files:
-    logging.error(
-        'Please select at least one option to begin scan\
- -k/--sa-key-path,-g/--gcloud-profile-path, -m, -rt, -at'
-    )
+  args = arguments.arg_parser()
 
   force_projects_list = list()
   if args.force_projects:
     force_projects_list = args.force_projects.split(',')
 
-  logging.basicConfig(level=getattr(logging, args.log_level.upper(), None),
+  # logging to a directory functionality
+  if args.log_directory and args.log_directory[0] == '/':
+    args.log_directory = args.log_directory[1:]
+
+  outputDir = f'{args.log_directory}' if args.log_directory else None
+  outputFile = f'{args.log_directory}/logs.log' if args.log_directory else None
+
+  if outputDir:
```

Comment:
We can capture that in the argument description that it will create a directory/directories for the path provided.